[Flaky Test] TestNacksTracker

https://github.com/apache/pulsar-client-go/actions/runs/4946174480/jobs/8843893910?pr=1015#step:5:8741

```
=== RUN   TestNacksTracker
    negative_acks_tracker_test.go:102: 
        	Error Trace:	/pulsar/pulsar-client-go/pulsar/negative_acks_tracker_test.go:102
        	Error:      	Not equal: 
        	            	expected: 2
        	            	actual  : 0
        	Test:       	TestNacksTracker
--- FAIL: TestNacksTracker (0.40s)
```

--- a/pulsar/negative_acks_tracker_test.go
+++ b/pulsar/negative_acks_tracker_test.go
@@ -42,10 +42,11 @@ func newNackMockedConsumer(nackBackoffPolicy NackBackoffPolicy) *nackMockedConsu
 	go func() {
 		// since the client ticks at an interval of delay / 3
 		// wait another interval to ensure we get all messages
+		// And we wait another 100 milliseconds to reduce the flaky
 		if nackBackoffPolicy == nil {
-			time.Sleep(testNackDelay + 101*time.Millisecond)
+			time.Sleep(testNackDelay + 200*time.Millisecond)
 		} else {
-			time.Sleep(nackBackoffPolicy.Next(1) + 101*time.Millisecond)
+			time.Sleep(nackBackoffPolicy.Next(1) + 200*time.Millisecond)
 		}
 		t.lock.Lock()
 		defer t.lock.Unlock()

